content/en/workflows/build.md

```diff
@@ -149,6 +149,37 @@ All the variables of the Source object are visible in the Context Variables tab.
 
 {{< img src="workflows/context-variables-tab-source-object-variables.png" alt="The Source object variables in the Context Variables tab" >}}
 
+## Error handling and fallbacks
+
+You can specify the number of times you want your workflow to retry a step and at what interval, before moving on to an optional fallback step. If no fallback step is provided, the workflow terminates after all retries have been exhausted.
```

Comment:
I would something like: "In case an action is failing when it runs, you can specify the number of times..."